src/common/runtime/server.ts

```diff
@@ -38,10 +39,22 @@ function usage(rc: number): never {
 interface RunResult {
   status: string;
   message: string;
+  coverageData?: string;
```

Comment:
Maybe explain that this is the path to the coverage data file and not the coverage data itself.

Reply:
Added a comment to make clear that it's opaque